specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -7892,6 +7892,14 @@
           "description": "The fabric specific object Id of the virtual machine.",
           "type": "string"
         },
+        "initialPrimaryFabricLocation": {
+          "description": "The initial primary fabric location.",
+          "type": "string"
+        },
+        "initialRecoveryFabricLocation": {
```

Comment:
this also looks like readonly

Reply:
It is a read only property. Made appropriate changes.